c++/triqs_cthyb/parameters.hpp

```diff
@@ -237,6 +241,9 @@ namespace triqs_cthyb {
 
     /// Threshold below which which off diagonal components of hloc are set to 0
     double off_diag_threshold = 0.0;
+    
+    /// Quadratic part of the local Hamiltonian. Must be provided if from_Delta = true
+    std::optional<std::vector<matrix<dcomplex>>> Delta_infty = std::optional<std::vector<matrix<dcomplex>>>{};
```

Comment:
```suggestion
    std::optional<many_body_op_t> h_loc0 = {};
```
I think that providing a many-body-operator is a more practical API than having to provide the `vector<matrix>`, what do you think?

This would require writing a separate function that converts the `h_loc0` into a `vector<matrix>`

Reply:
I'm not sure I agree. 

Currently Solver.Delta_infty gives a vector of matrices (or list of arrays in python) so I thought it would be better to have the input be the same format.